com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UNetTransport.cs

```diff
@@ -399,6 +399,8 @@ public ConnectionConfig GetConfig()
         {
             var connectionConfig = new ConnectionConfig();
 
+            connectionConfig.FragmentSize = 1024;
```

Comment:
is this new or was it already been there before?

Reply:
I mentioned in the description - FragmentSize defaults to 500. Our batcher is batching at 512. Which means almost every single batch we send gets fragmented. Either FragmentSize needs increased, or batch size needs decreased... one of the two.